app/services/weetsService.js

```diff
@@ -0,0 +1,15 @@
+const request = require('../utils/request');
+
+const weetsService = {
+  getRandomNumber() {
```

Comment:
i needed it because i did this.getRandomNumber but also i can do weetsService.getRandomNumber

Reply:
if it is in the same scope you do not need to call it as an object method. it could be an static method in the context. 
Also, if you need to export it, then it is not a valid weetService method. In that case, it would be better to have it in a utils or helper